packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfCustomControlLibrary-VisualBasic/CustomControl1.vb

```diff
@@ -0,0 +1,21 @@
+Imports System.Windows
+Imports System.Windows.Controls
+
+Public Class CustomControl1

```

Comment:
I was going to suggest that this needs to be enclosed in a namespace, but the .NET Framework version does not have a namespace either. 

Oddly enough, the `Import`s are missing in the .NET Framework versions, but they are clearly required in .NET Core.